xarray/tests/test_coding_times.py

```diff
@@ -671,6 +672,48 @@ def test_decode_cf_time_bounds():
     _update_bounds_attributes(ds.variables)
 
 
+def test_encode_time_bounds():
```

Comment:
It looks like you'll need a `@requires_cftime_or_netCDF4` decorator here to make sure this test gets skipped without one of those installed.